### DISCUSSION TITLE
Disable scrolling to next page?

### DISCUSSION THREAD

**Discussion by Lobuttomize** [sumatrapdfreader/sumatrapdf#2490] [Q&A] [answered]
Is it currently possible (in the pre-release) to disable scrolling to the next page? When using it for CBZ, I want to hit a limit at the bottom of each page until I hit the button specifically to turn the page. In simple terms, if you were on page 3 and held the spacebar down, it would never advance to page 4; it would only stay on the bottom of page 3.

**Comment by kjk** [comment, marked as answer]
No, there is no such functionality.

**Reply by GitHubRulesOK** [reply]
@kjk ***Hmmm*** officially you are clearly right
 but if I use new reassign Space to a function not valid in CBX the spacebar is non functional in CBX 🙂 

Shortcuts [
	[
		Cmd = CmdCreateAnnotText
		Key = Space
	]
]

But thats clearly not @Lobuttomize s intention.

**Reply by kjk** [reply]
Tip: to disable a key, assign to `CmdNone`.

I'm not sure if disabling `Space` would work the way @Lobuttomize wants but I might be wrong.
